conf/master

```diff
@@ -181,8 +184,9 @@
 #ipc_mode: ipc
 
 # Overwrite the default tcp ports used by the minion when ipc_mode is set to 'tcp'
-#tcp_master_pub_port: 4510
-#tcp_master_pull_port: 4511
+#tcp_master_pub_port: 4512
+
```

Comment:
i don't like adding this newline

Reply:
As per comment at the top of the `conf/master` file:

    # This configuration file is used to manage the behavior of the Salt Master.
    # Values that are commented out but have an empty line after the comment are
    # defaults that do not need to be set in the config. If there is no blank line
    # after the comment then the value is presented as an example and is not the
    # default.

This is to highlight disambiguate option which **are** the default - otherwise, the file is all over the place and without further reading, user has no way of telling whether a particular option is the default or not. Please see #50025 what prompted both the issue and this PR.